dev_guide/compute_resources.adoc

```diff
@@ -137,3 +137,11 @@ Containers:
     Environment Variables:
 ----
 ====
+
+== Project Resource Limits
+
+link:../admin_guide/limits.html-limits[Resource limits can be
```

Comment:
s/-/#/

We also have to consider the openshift-dedicated and openshift-online distros, which don't include the admin_guide and install_config, therefore breaking these links in those builds. We've generally used ifdefs for this sort of situation.